ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -2218,6 +2218,24 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: baremetal-lab-agent-install-network-static-disconnected
+- as: baremetal-3-plus-1-agent-ipv4-static-p2-f14
+  cluster: build05
+  cron: 1 6 3,18 * *
+  steps:
+    cluster_profile: equinix-ocp-metal-qe
+    env:
+      AUX_HOST: openshift-qe-bastion.arm.eng.rdu2.redhat.com
+      E2E_RUN_TAGS: '@baremetal-upi and @amd64 and @network-ovnkubernetes and not
+        @fips'
+      RESERVE_BOOTSTRAP: "false"
+      architecture: amd64
+      ipv4_enabled: "true"
+      ipv6_enabled: "false"
+      masters: "3"
+      workers: "1"
+    test:
+    - chain: openshift-e2e-test-qe
+    workflow: baremetal-lab-agent-install-network-static
```

Comment:
Can we please create a workflow dedicated to `3+1` cluster installation? We can override the environment variables from the workflow and remove the following from here:

```
      ipv4_enabled: "true"
      ipv6_enabled: "false"
      masters: "3"
      workers: "1"
```


I make this suggestion because I want to expose minimal environment variables and options to the job in order to  eliminate the maintenance efforts from someone that do not understand the the use of these variables.

Reply:
The changes have been implemented.